### ISSUE TITLE
Option to customize bitrate

### ISSUE THREAD

**Issue by dxzdxz1** [mtolly/onyxite-customs#168]
tl;dr
Is it possible to include a setting to choose what approximate quality/bitrate I want for my customs?

I used to play on Phase Shift but I'm migrating to an used X360 that I bought recently, so I'm re-converting all of my songs

I've been using [Spleeter](https://github.com/deezer/spleeter)  to create separate stems of all instruments and it has been working brilliantly so far. I always use a flac as source, separate all stems and them I convert to ogg VBR q6 (aprox 192kbps) for each stem. It keeps everything at high quality, and yet compatible with PS.

I know that MagmaC3 has an option to choose what bitrate I want to use, _and I can also create a multi-channel ogg using Audacity, put a mogg header onto the file and replace the song from the con using Le Fluffie_, but your program is so much easier to handle and has an option to do everything as a batch process, so I'm wondering if is possible to you to create this option too

**Comment by mtolly** [comment]
Hey, you are in luck :) I added an Ogg Vorbis compression control to the preferences screen just recently, and it should go out in a release pretty soon once I get everything tested and built. I'll message here when it is up.

**Comment by mtolly** [comment]
Release `20210110` is up with the compression level slider in Preferences.
